blockbase/sass/blocks/_navigation.scss

```diff
@@ -22,6 +22,7 @@
 }
 
 .wp-block-navigation.is-style-blockbase-navigation-improved-responsive {
+	flex-grow: 1;
```

Comment:
yeah, the flex grow breaks the editor, I didn't refactor the block style for the navigation, it would need re testing

Reply:
flex-grow is needed so that the navigation aligns right when it wraps

Reply:
it's not needed anymore with this PR's approach, we wrap logo, site title and tagline in a group, use the editor's "space between" rule to align correctly without using flex-grow, and then use display contents on the wrapper we added for everything to work fine in mobile. It can be refactored, I just forgot we had the block style altogether

Reply:
Without this rule the navigation looks like this when it wraps
<img width="945" alt="Screenshot 2021-12-01 at 18 05 03" src="https://user-images.githubusercontent.com/275961/144289078-cbfa024d-16a6-4c07-93d8-4d8e7d5bb9d1.png">
:

Reply:
After thinking about this a bit...

If that's the behavior of blocks in a row block when alignment is set to "space between" then I don't know that we should mess with it with the `flex-grow` attribute.

It looks funny (wrong) EITHER way when the menu wraps.  My opinion is then when wrapped right-aligned looks MORE wrong.  But looks isn't why I think we should do away with the `flex-grow` but rather the behavior.  If we don't like it then we should open a Gutenberg issue regarding the wrap behavior of rows using "space between".

Reply:
I agree this is the best solution. I think its unlikely anyone will notice.

Reply:
yeah, sorry @scruffian I didn't know you meant about the flex-grow when wrapping (we used it initially to make the layout actually work, else the menu would always move to the left no matter what). I think I agree with Jason on this too.